AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m

```diff
@@ -29,6 +29,13 @@ + (instancetype)sharedDownloader
 
 #pragma mark ASImageProtocols
 
+- (UIImage *)synchronouslyFetchedCachedImageWithURL:(NSURL *)URL
+{
+  NSString *key = [[PINRemoteImageManager sharedImageManager] cacheKeyForURL:URL processorKey:nil];
```

Comment:
Incredibly minor, but local variable for sharedImageManager makes sense for this - probably immesurable since the iPhone 3G -> 3GS, armv6 -> armv7 switch though :-P